### DISCUSSION TITLE
I can't find NuxtI18nInstance in v8.

### DISCUSSION THREAD

**Discussion by atlansien** [nuxt-modules/i18n#2087] [Q&A]
### Describe the feature

The `NuxtI18nInstance` released in [v6.25.0](https://github.com/nuxt-modules/i18n/releases/tag/v6.25.0) is missing in v8. What should I use instead?

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution) (The contribution guideline of nuxt-modules/i18n is compliant with Nuxt too).
- [X] Check existing [discussions](https://github.com/nuxt-modules/i18n/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by kazupon** [comment]
This issue is question, So we will transfer to github discussion.

**Reply by kazupon** [reply]
You can use `$i18n` via `useNuxtApp`
https://nuxt.com/docs/guide/going-further/nuxt-app#accessing-nuxtapp

**Reply by atlansien** [reply]
@kazupon 
返信ありがとうございます！自分最初は`typeof $i18n` で対応していました。
ただライフサイクル外ですと `useNuxtApp` が使えないため質問させていただたいた次第です🙇

**Reply by kazupon** [reply]
@atlansien 
ライフサイクル外で利用したいとのことですが、具体的にどういったケースでしょうか？

**Reply by atlansien** [reply]
@kazupon 
`utils` で i18n を引数とした関数を作成したので、引数に型を当てたいと思った次第です。
